pages/home/auth-providers/atlassian.mdx

```diff
@@ -27,8 +27,8 @@ At the moment, you can only configure Atlassian auth with a [self-hosted Engine]
 
 - Create a Atlassian app in the [Atlassian developer console](https://developer.atlassian.com/console/myapps/)
 - In the Authorization tab, click the "Add" action button and set the Callback URL to: `https://cloud.arcade.dev/api/v1/oauth/callback`
-- In the Permissions tab, set any permissions you need for your app
-- In the Settings tab, copy the Client ID and Secret to use later
+- In the Permissions tab, enable any permissions you need for your app
+- In the Settings tab, copy the Client ID and Secret to use below
```

Comment:
Small wording change on every page to make it clear that the client ID/secret is used in the very next step